Documentation/ceph-csi-drivers.md

```diff
@@ -32,19 +32,12 @@ cluster.
 kubectl get ns
 kubectl create namespace rook-ceph
```

Comment:
the namespace will be created as part of common.yaml we could have removed the manual namespace section